apache/incubator-zeppelin
# Add tachyon interpreter

### What is this PR for?

Add an interpreter to work with a tachyon file system.
Properties required for the interpreter are:

* tachyon master hostname
* tachyon master port

### What type of PR is it?

feature

### Is there a relevant Jira issue?

https://issues.apache.org/jira/browse/ZEPPELIN-604

### How should this be tested?

* [Install and configure Tachyon](http://tachyon-project.org/downloads/) on a local machine.
* run Tachyon in local mode ```$ ./bin/tachyon-start.sh local```
* check that interpreter params are setted to default values (hostname: localhost, port: 19998)
* use the [Tachyon CLI commands](http://tachyon-project.org/documentation/Command-Line-Interface.html) to interact with your Tachyon file system

### Screenshots (if appropriate)

![tachyon-interpreter](https://cloud.githubusercontent.com/assets/7325722/12291741/5b145628-b9e9-11e5-98a5-fd33966da96a.png)

### Questions:

* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

/cc @jsimsa for the support he give us to develop this feature
